corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -34,5 +34,34 @@ hqDefine("scheduling/js/conditional_alert_main", [
         // setup tab
         basicInformation.setRuleTabVisibility();
         $("#conditional-alert-basic-info-panel").koApplyBindings(basicInformation);
+
+        $("#conditional-alert-save-btn").click(function () {
+            var newRuleName = $("#id_conditional-alert-name").val();
+            if (!name) {
+                $("#conditional-alert-form").submit();
+            } else if (name !== newRuleName) {
+                var caseType = $(
+                    "#select2-id_criteria-case_type-container"
+                ).text();
+                var caseCountUrl = $("#conditional-alert-case-count-url").attr(
+                    "url"
+                );
+                $.ajax({
```

Comment:
This is going to cause a delay, so we should signal to the user that HQ is working so they don't repeatedly click save. The usual way we'd do this is disable the button and show a spinner on it, then re-enable the button when the ajax call is complete. There are `disableButton` and `enabledButton` functions to assist with this.